cs3api4lab/tests/test_cs3apismanager.py

```diff
@@ -209,37 +201,41 @@ def _create_notebook_model(self):
     def test_delete_file(self):
         file_path = "/test_delete_exits_file.txt"
         message = "Lorem ipsum dolor sit amet..."
-        self.file_api.write_file(file_path, message, self.endpoint)
-
-        self.contents_manager.delete_file(file_path)
+        try:
+            self.file_api.write_file(file_path, message, self.endpoint)
+        finally:
+            self.contents_manager.delete_file(file_path)
 
         with self.assertRaises(IOError):
```

Comment:
this should be inside try catch

Reply:
why?

Reply:
if w delete the file in finally block then assertRaises here will always pass